src/main/java/dummydata/BookOwner.java

```diff
@@ -0,0 +1,34 @@
+package dummydata;
+
+import model.Book;
+import model.User;
+import model.UserBuilder;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static dummydata.UserSample.sampleReadyUser;
+import static dummydata.UserSample.userData;
+
+public class BookOwner {
```

Comment:
Book owner is a single class or collection is not clear from the behaviour of the class.